py-polars/polars/internals/expr.py

```diff
@@ -382,14 +384,7 @@ def prefix(self, prefix: str) -> "Expr":
         ├╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌┼╌╌╌╌╌┼╌╌╌╌╌╌╌╌╌╌┤
         │ 5   ┆ "banana" ┆ 1   ┆ "beetle" │
         ╰─────┴──────────┴─────┴──────────╯
-        >>> (
-        ...     df.select(
-        ...         [
-        ...             pl.all(),
-        ...             pl.all().reverse().suffix("_reverse"),
-        ...         ]
-        ...     )
-        ... )
+        >>> (df.select([pl.all(), pl.all().reverse().suffix("_reverse")]))
```

Comment:
@ghuls : for my setup, blackdoc suggested this one-liner

Reply:
I believe @ghuls suggested ending the final argument with `,` would help turn it into a multiline docstring. Not sure though.

Reply:
Would we want that in the first place? I.e., I actually think this is clearer on a single line, in particular if the first and last bracket, which arent needed, are removed?

Reply:
first and last bracket can indeed be removed here, but I think @ritchie46 added them before for other samples to be able to do put `.expr` at a new line:
```python
(df
 .select(
   [...]
 )
)
````

I think for most expressions, it makes sense to have them spread over multiple lines, to make it much easier to comprehend the queries.

Reply:
I do personally think the additional brackets make it more cumbersome, lots of vertical space being used. Agreed on multi-line being more readable in various places, I will revert those and see if blackdoc is happy with it (I was kind of hoping blackdoc would avoid all these discussions, but it does seem it is happy with multiple formats).

Reply:
@zundertj Are you still working on this branch? Seems like there are things in here that I already fixed.

Also you have changes like `pl.col` to `col` (even if it is imported as `col` in that file, I think using `pl.col` in the documentation is clearer.

Another option is to run blackdoc manually and fix some stuff to make it more compact. Doctest probably will change less than normal code, so the burden might not be super high.

Reply:
Yes, I am, was a bit of a struggle to merge in your changes already, but that should be good now, as there are no merge conflicts.

Reply:
Found one more place where it should `pl.col` indeed, should now be good.

Reply:
there was also at least one `Dataframe` instead of `pl.Dataframe` (not sure if it is still in the latest code). (I also fixed some `lit` ==> `pl.lit`

Reply:
Note that your fixes wont show up unless I would be rolling them back here. Is that the case?